pros/examples/accessories.rs

```diff
@@ -6,10 +6,21 @@ use pros::prelude::*;
 
 #[derive(Debug, Default)]
 struct ExampleRobot;
+#[async_trait]
```

Comment:
Thoughts on waiting for async traits to stabilize?

Reply:
I was thinking about that, but it might be a while until then. The only real upside to waiting is that we will have a slightly better API on release, but it is better, in my opinion, to get async out earlier than later because of how much it improves things. It would also not be a very hard change to make later.